mysql/cloudmysql/cloudmysql.go

```diff
@@ -56,7 +56,7 @@ type Params struct {
 }
 
 // Open opens a Cloud SQL database.
-func Open(ctx context.Context, certSource proxy.CertSource, params *Params) (*sql.DB, error) {
+func Open(ctx context.Context, certSource proxy.CertSource, params *Params, traceOpts ocsql.TraceOption) (*sql.DB, error) {
```

Comment:
It looks like ocsql.TraceOption is a func; what you have here only allows a single one. I.e., what if I want WithRowsAffected and WithRowsClosed?

Maybe this should be a *ocsql.TraceOptions instead?

Reply:
Fixed it by making it varargs like it's supposed to be.